common/src/main/java/com/wynntils/overlays/StatusEffectsOverlay.java

```diff
@@ -138,52 +140,93 @@ private Stream<RenderedStatusEffect> stackEffects(List<StatusEffect> effects) {
         Map<String, RenderedStatusEffect> effectsToRender = new LinkedHashMap<>();
 
         for (StatusEffect effect : effects) {
-            RenderedStatusEffect entry = effectsToRender.get(effect.asString().getString());
+            String key = getEffectsKey(effect);
+            RenderedStatusEffect entry = effectsToRender.get(key);
 
             if (entry == null) {
                 entry = new RenderedStatusEffect(effect);
-                effectsToRender.put(effect.asString().getString(), entry);
+                effectsToRender.put(key, entry);
             }
 
             entry.setCount(entry.getCount() + 1);
+
+            if( effect.hasModifierValue() ){
+                entry.addModifier(effect.getModifierValue());
+            }
         }
 
         return effectsToRender.values().stream();
     }
 
-    private static final class RenderedStatusEffect {
+    private String getEffectsKey(StatusEffect effect) {
+        return switch (effectsStackBehaviour.get()) {
+            default -> effect.asString().getString();
+            case SUM -> effect.getPrefix().getString()
+                    + effect.getName().getString()
+                    + (effect.getModifier().getString().indexOf('%') == -1)
```

Comment:
You want to add a true/false to the string? I guess it would work but this is weird.

Reply:
It's simply a check for if the effect is % based or not. Do you know of a different method that would work?

Reply:
Is this something that should be changed or is it fine as is?

Reply:
Why not just append the modifier?

Reply:
Why not just append the modifier?